scripts/wasmd/contracts/checksums.sha256

```diff
@@ -1,4 +1,4 @@
-775f9f8fe7d2ca1ac33ec47799e9b8fd25aa1bf3cf97961f49f091b99e62f85b  cw1_subkeys.wasm
-3e3ea92c4ec35d90aa0bef344bdb0a94939120fce57265d5ff81bf8e8caec7e9  cw3_fixed_multisig.wasm
-a32acdcfe15a2b3c8ba6963cf1e4ab63347725cc35a0f2434696dd492d63fb5f  hackatom.wasm
-089598e3cab468aff26233bc7363c713529124a0f7fd50320461cb58881a2da7  ibc_reflect.wasm
+37f868eda7a723a769f6ff77cc27435455c3cff7b14481a114895d3d93cf8326  cw1_subkeys.wasm
```

Comment:
This is based on v0.8.0-rc1 and v0.16.0-rc5, right?

Do you need a final cosmwasm-plus release before you can cut cosmjs? We are not changing the interfaces of those contracts since the versions you use

Reply:
Right. I don't need anything else from cosmwasm-plus. Those contracts are not used right now.